locales/zh_CN/LC_MESSAGES/hello.po

```diff
@@ -213,125 +210,117 @@ msgstr "（可用: 该系统上有最完整的支持；不可用: 由于平台
 "device_memory_GB=3.4)`` 来分配 ``3.4`` GB 显存，或者使用 ``ti.init(arch=ti.cuda, "
 "device_memory_fraction=0.3)`` 来分配所有可用显存的 ``30%``."
 
 #: ../../hello.rst:102
-#, fuzzy
 msgid ""
-"On other platforms, Taichi will make use of its on-demand memory "
-"allocator to adaptively allocate memory."
-msgstr "在其他平台上， Taichi 会使用自适应内存分配器来动态地分配内存。"
+"On other platforms, Taichi will make use of its on-demand memory allocator to adaptively allocate "
+"memory."
+msgstr "在其他平台上， Taichi 将会使用它的自适应内存分配器来动态分配内存。"
 
 #: ../../hello.rst:105
 msgid "(Sparse) tensors"
 msgstr "（稀疏）张量"
 
 #: ../../hello.rst:107
-#, fuzzy
 msgid ""
-"Taichi is a data-oriented programming language where dense or spatially-"
-"sparse tensors are the first-class citizens. See :ref:`sparse` for more "
-"details on sparse tensors."
-msgstr "Taichi是一门面向数据的编程语言，（稠密、稀疏）张量是一等公民。欲了解更多关于稀疏张量的内容，请见 :ref:`sparse`。"
+"Taichi is a data-oriented programming language where dense or spatially-sparse tensors are the first-"
+"class citizens. See :ref:`sparse` for more details on sparse tensors."
+msgstr ""
+"Taichi 是一门面向数据的程序设计语言，其中（稠密、稀疏）张量是第一类公民(First-class Citizen)。在 :ref:"
+"`sparse` 这一章节，你可以了解到更多关于稀疏张量的详细信息。"
 
 #: ../../hello.rst:110
 msgid ""
-"In the code above, ``pixels = ti.var(dt=ti.f32, shape=(n * 2, n))`` "
-"allocates a 2D dense tensor named ``pixels`` of size ``(640, 320)`` and "
-"element data type ``ti.f32`` (i.e. ``float`` in C)."
+"In the code above, ``pixels = ti.var(dt=ti.f32, shape=(n * 2, n))`` allocates a 2D dense tensor named "
+"``pixels`` of size ``(640, 320)`` and element data type ``ti.f32`` (i.e. ``float`` in C)."
 msgstr ""
-"在以上代码中，``pixels = ti.var(dt=ti.f32, shape=(n * 2, n))`` 分配一个叫做 ``pixel`` "
-"的二维张量，大小是 ``(640, 320)`` ，数据类型是 ``ti.f32`` (也即C语言的 ``float``)."
+"在以上代码中，``pixels = ti.var(dt=ti.f32, shape=(n * 2, n))`` 分配了一个叫做 ``pixels`` 的二维张量，"
+"大小是 ``(640, 320)`` ，数据类型是 ``ti.f32`` （即，C语言中的 ``float``）."
 
 #: ../../hello.rst:114
 msgid "Functions and kernels"
 msgstr "函数与内核"
 
 #: ../../hello.rst:116
-#, fuzzy
 msgid ""
-"Computation resides in Taichi **kernels**. Kernel arguments must be type-"
-"hinted. The language used in Taichi kernels and functions looks exactly "
-"like Python, yet the Taichi frontend compiler converts it into a language"
-" that is **compiled, statically-typed, lexically-scoped, parallel and "
-"differentiable**."
+"Computation resides in Taichi **kernels**. Kernel arguments must be type-hinted. The language used in "
+"Taichi kernels and functions looks exactly like Python, yet the Taichi frontend compiler converts it "
+"into a language that is **compiled, statically-typed, lexically-scoped, parallel and differentiable**."
 msgstr ""
-"Taichi的计算是在 **内核** "
-"中进行的。内核的参数必须显式指定类型。Taichi内核与函数中所用的语法，看起来和Python的很像，Taichi的前端编译器会将其转换为 "
-"**编译型，静态类型，有词法作用域，并行执行，可微分** 的语言。"
+"计算发生在 Taichi 的 **内核(kernel)** 模块中。内核的参数必须显式指定类型。Taichi 内核与函数中所用的语"
+"法，看起来和 Python 的很像，然而 Taichi 的前端编译器会将其转换为 **编译型，静态类型，有词法作用域，并"
+"行执行，可微分** 的语言。"
```

Comment:
```suggestion
"计算发生在 Taichi 的 **内核(kernel)** 中。内核的参数必须显式指定类型。Taichi 内核与函数中所用的语"
"法，看起来和 Python 的很像，然而 Taichi 的前端编译器会将其转换为 **编译型，静态类型，有词法作用域，并"
"行执行，可微分** 的语言。"
```
`kernel` is not a Taichi module at all. In Taichi, kernels are just a special case of user-defined functions.

Reply:
I know it, and the idea is that if we can keep the ``kernel`` and ``functions`` in English it might be easier for the reader to establish a relationship with the code.So I added the word ``模块`` at the beginning with this goal in mind.
 It seems that this behavior will eventually lead to some misunderstanding 🧐